Desktop/MyBudgetWindow.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Diagnostics;
+using System.Drawing;
+using System.Text;
+using System.Windows.Forms;
+using SmartSaver.Data;
+using SmartSaver.Models;
+
+namespace SmartSaver.Desktop
+{
+    public partial class MyBudgetWindow : Form
+    {
+        private Database db = new Database();
+        private List<Category> CategoryList;
+        public MyBudgetWindow()
+        {
+            InitializeComponent();
+        }
+
+        private void MyBudgetWindow_Load(object sender, EventArgs e)
+        {
+            UpdateCategoriesList();
+        }
+
+        public void UpdateCategoriesList()
+        {
+            CategoryList = db.GetCategories();
+            PopulateCategoryListView();
+        }
+
+        private void PopulateCategoryListView()
+        {
+            PopulateCategoryListView(CategoryList);
+        }
```

Comment:
What is the point of this method?

Reply:
To get data from the database and display it in a window.

Reply:
But why do you not just call the method in line 39 from line 31?